predator/values.yaml

```diff
@@ -63,3 +63,7 @@ runnerMemory: 256
 intervalCleanupFinishedContainers: 0
 customRunnerDefinition:
 allowInsecureTls: true
+serviceAccount:
```

Comment:
I would keep this both by default as true for easy installation 
but it's good idea to toggle the off